pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1336,6 +1337,14 @@ public void onUpdate(String path, Map<String, String> data, Stat stat) {
         return dynamicConfigurationMap.keys();
     }
 
+    public List<String> getRuntimeConfiguration() {
+        ConcurrentOpenHashMap<String, Object> runtimeConfigurationMap = getRuntimeConfigurationMap();
+        return runtimeConfigurationMap.keys().stream()
+            .sorted()
+            .map(key -> key + "=" + runtimeConfigurationMap.get(key))
```

Comment:
The configuration is already logically a map. By flattening to a list of `key=value` we're going to leave any user of this API to break it down again.

I'd prefer the API to just return a `Map<String, String>` instead of this `List<String>` with encoded values.

Reply:
Changed, thanks!